f3/granite.go

```diff
@@ -73,6 +73,8 @@ type instance struct {
 	// State for each round of phases.
 	// State from prior rounds must be maintained to provide justification for values in subsequent rounds.
 	rounds map[int]*roundState
+	// Heaviest acceptable chain
+	heaviestChain ECChain
```

Comment:
I would just call this "acceptable". The granite code never looks at the weight or compares it to another chain. The "heaviest" nature is something external. For this code, it's simply a predicate of what it will accept.